cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/datastore/Datastore.java

```diff
@@ -49,7 +48,7 @@
  * propagated, thereby resolving the conflicted document across the set of
  * peers.</p>
  *
- * @see DatastoreExtended
+ * @see DatastoreImpl
```

Comment:
I don't think we can pretend it doesn't exist. Note that this documentation is also aimed at library developers as well as library consumers. Or do you think DatastoreImpl should have a stronger warning to tell library consumers not to use it directly?